lua/gitblame/init.lua

```diff
@@ -39,9 +39,6 @@ local date_format_has_relative_time
 ---@type string
 local current_blame_text
 
----@type table timer luv timer object
-local delay_timer
-
```

Comment:
This was removed in the most recent commits (https://github.com/f-person/git-blame.nvim/commit/62a39ca5a86f311fa4fbfff7e57a8f2c8e197ed5)